volume/service/service.go

```diff
@@ -73,6 +73,9 @@ func (s *VolumesService) Create(ctx context.Context, name, driverName string, op
 	if name == "" {
 		name = stringid.GenerateRandomID()
 		options = append(options, opts.WithCreateLabel(AnonymousLabel, ""))
+		log.G(ctx).WithField("volume-name", name).Error("Creating anonymous volume")
+	} else {
+		log.G(ctx).WithField("volume-name", name).Error("Creating named volume")
```

Comment:
Whoops; I left these as `Error()` log instead of `Debug()` 🙈 

- https://github.com/moby/moby/pull/48767